solutions/devsprint-bruno-ramos-5/GitHubApp/Screens/List/ListView.swift

```diff
@@ -12,6 +12,10 @@ struct ListViewConfiguration {
     let listItems: [String]
 }
 
+protocol ListViewDelegate {
```

Comment:
Seguindo esse comentário aqui https://github.com/devpass-tech/challenge-github-app/pull/148/files#r918383658 e já peço desculpas pelo meu não entendimento, acredito que esse protocolo não seja um delegate mesmo (mals pelo vacilo), mas sim o protocolo de comunicação da view controller com a view.
```suggestion
protocol ListViewProtocol {
```

Reply:
não precisa se desculpar não heheheh.... a sua ideia era ajudar :)
 já valeu pela discussão... tmj professor